### PR TITLE
Add tox and travis support

### LINKED ISSUE #3: Add tox and travis
We should add travis and tox to ensure test pass on all the suported platforms (py27, py34, py35)

I'm wondering if we should also add coverage and pep8 checks

I can work on it

### PR DESCRIPTION
**Pull request by mbohlool**
fixes #3

based on #6 by @pokoli

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[tox]
+skipsdist = True
+envlist = py
+
+[testenv]
```
Comment (pokoli) [inline tox.ini]: What about coverage?

**Comment by mbohlool** [comment]
@pokoli this is ready now. Thanks for the review.

File: api_client.py [hunk 2]
```diff
@@ -1,45 +1,38 @@
-# coding: utf-8
-
-"""
-Copyright 2016 SmartBear Software
```
Comment (pokoli) [inline api_client.py]: Are we sure we can remove this copy right?

File: run_tox.sh [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+SCRIPT_ROOT=`pwd`
+popd > /dev/null
+
+cd "${TMP_DIR}"
+git clone https://github.com/kubernetes-incubator/client-python.git
+cd client-python
+git config user.email "kubernetes-client@k8s.com"
+git config user.name "kubenetes client"
+git rm -rf kubernetes/base
+git commit -m "DO NOT MERGE, removing submodule for testing only"
```
Comment (pokoli) [inline run_tox.sh]: I'm wondering why is required to commit, can we avoid it?

**Comment by pokoli** [comment]
@mbohlool Looks good to me. I added two minor comments, but for me it can be merged. Good work.

File: run_tox.sh (hunk 3 above)
Comment (mbohlool) [inline run_tox.sh]: Update-pep8 check will fail if there is an uncommited change.

File: api_client.py (hunk 2 above)
Comment (mbohlool) [inline api_client.py]: Good catch. I prefer not to remove that. Will fix.

File: run_tox.sh (hunk 3 above)
Comment (pokoli) [inline run_tox.sh]: Acknowledged. I was not aware of this part! Great.

File: tox.ini (hunk 1 above)
Comment (mbohlool) [inline tox.ini]: All environment from .travis file will be passed to the main repo tox, coverage is running too (codecov is already commenting on the PR).

**Comment by pokoli** [comment]
LGTM

**Pull request merged**